examples/text_labels_and_annotations/unicode_minus.py

```diff
@@ -3,25 +3,18 @@
 Unicode minus
 =============
 
-You can use the proper typesetting `Unicode minus`__ or the ASCII hyphen for
-minus, which some people prefer.  :rc:`axes.unicode_minus` controls the default
-behavior.
+By default, tick labels at negative values are rendered using a `Unicode
+minus`__ (U+2212) rather than an ASCII hyphen (U+002D).  This can be controlled
+by setting :rc:`axes.unicode_minus` (which defaults to True).
```

Comment:
This sentence seems incomplete:
```suggestion
by setting :rc:`axes.unicode_minus` (which defaults to True) to False.
```

Reply:
"It can be controlled by setting the rcParam (meaning, to either value)"?

Reply:
I think it is fine the way it is.